src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -28,6 +28,12 @@ Microsoft.CodeAnalysis.CSharp.SyntaxKind.MultiLineRawStringLiteralExpression = 9
 Microsoft.CodeAnalysis.CSharp.SyntaxKind.MultiLineRawStringLiteralToken = 9073 -> Microsoft.CodeAnalysis.CSharp.SyntaxKind
 Microsoft.CodeAnalysis.CSharp.SyntaxKind.SingleLineRawStringLiteralExpression = 9074 -> Microsoft.CodeAnalysis.CSharp.SyntaxKind
 Microsoft.CodeAnalysis.CSharp.SyntaxKind.SingleLineRawStringLiteralToken = 9072 -> Microsoft.CodeAnalysis.CSharp.SyntaxKind
+Microsoft.CodeAnalysis.CSharp.SyntaxKind.SingleLineRawInterpolatedStringStartToken = 9080 -> Microsoft.CodeAnalysis.CSharp.SyntaxKind
```

Comment:
>SingleLineRawInterpolatedStringStartToken [](http://example.com/codeflow?start=41&length=41)

nit: we may want ...InterpolatedRawString... instead of ...RawInterpolatedString..., as we have `InterpolatedVerbatimStringStartToken`

Reply:
Very good point!

Reply:
You may want to run this by compiler API review before making changes to the PR, to reduce risk of rework.